views/pages/infoAcolhido.ejs

```diff
@@ -7,8 +7,8 @@
         <%- include partials/cabecalhoTelas.ejs %>
     </div>
     <div class="botoes medicamento">
-        <form action="/acolhido/<%= acolhido.id %>/prescricao" method="POST">
+        <form action="/acolhido/<%= acolhido.id %>/prescricao/create" method="GET">
```

Comment:
Essa rota deve se chamar `new` se a gente for seguir dessa forma. Você redireciona para `/new` com GET e ai quando for criar usa o `POST` para `/create`.